README.md

```diff
@@ -163,13 +171,27 @@ test('should have the correct attributes', () => {
 });
 ```
 
+You can now add test attributes in test titles using `@tag` notation. This provides a succinct way to include metadata in test reports. Any `@tag` in a jest test title will be used as an attribute in ReportPortal, but won't appear in the final test title on ReportPortal.
+
+Example:
+
+```javascript
+test('should have a tag at the beginning of the test title', () => {
+  ReportingApi.addAttributes([{ key: 'title', value: 'tag' }]);
```

Comment:
So please remove this method usage just to avoid mixing two approaches in this example.
```suggestion
```